src/assets/_scss/extends/_slider.scss

```diff
@@ -0,0 +1,97 @@
+%slider {
+  -webkit-appearance: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  &:focus {
+    outline: none;
+
+    &::-webkit-slider-runnable-track {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-webkit-slider-thumb {
+      border: 1px solid darken($color-gray-cool-light, 10);
+    }
+
+    &::-ms-fill-lower {
+      background: lighten($color-gray-cool-light, 3);
+    }
+
+    &::-ms-fill-upper {
+      background: lighten($color-gray-cool-light, 3);
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
+    background: lighten($color-gray-cool-light, 3);
+    border: 1px solid darken($color-gray-cool-light, 15);
+    border-radius: 0.5em;
+    cursor: pointer;
+    height: 0.5em;
+    width: 100%;
+  }
+
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    background: $color-white;
+    border: 1px solid darken($color-gray-cool-light, 10);
+    border-radius: 50%;
+    box-shadow: 0 2px 3px rgba($color-black, 0.4), 0 0 0 #0d0d0d;
```

Comment:
Color literals like `#0d0d0d` should only be used in variable declarations; they should be referred to via variable everywhere else.